tests/e2e/csi_snapshot_basic.go

```diff
@@ -2268,16 +2305,18 @@ var _ = ginkgo.Describe("Volume Snapshot Basic Test", func() {
 			"Number of Pods in the statefulset should match with number of replicas")
 
 		framework.Logf("Fetching pod 1, pvc1 and pv1 details")
+		// pod1 details
 		pod1, err := client.CoreV1().Pods(namespace).Get(ctx,
```

Comment:
Put in a loop for all replicas instead of creating one by one

Reply:
Acknowledged kavya's comment here where I am keeping only 1 stateful replica for snapshot verification